packages/exo/README.md

```diff
@@ -10,13 +10,19 @@ When an exo is defined with an InterfaceGuard, the exo is augmented by default w
 
 ```js
 // `GET_INTERFACE_GUARD` holds the name of the meta-method
-import { GET_INTERFACE_GUARD } from `@endo/exo`;
+import { GET_INTERFACE_GUARD } from '@endo/exo';
+import { getCopyMapKeys } from '@endo/patterns';
 
 ...
    const interfaceGuard = await E(exo)[GET_INTERFACE_GUARD]();
    // `methodNames` omits names of automatically added meta-methods like
    // the value of `GET_INTERFACE_GUARD`.
    // Others may also be omitted if `interfaceGuard.partial`
-   const methodNames = Reflect.ownKeys(interfaceGuard.methodGuards);
+   const methodNames = Reflect.ownKeys(interfaceGuard.methodGuards)
+     .concat(
+       interfaceGuard.symbolMethodGuards
+         ? getCopyMapKeys(interfaceGuard.symbolMethodGuards)
+         : []
+     );
```

Comment:
I like the following style a bit better, but up to you.

```suggestion
   const methodNames = [
     ...Reflect.ownKeys(interfaceGuard.methodGuards),
     ...interfaceGuard.symbolMethodGuards
         ? getCopyMapKeys(interfaceGuard.symbolMethodGuards)
         : [],
   ];
```

Reply:
Done.